### PR TITLE
Add tools to build Go programs during the build process

### PR DESCRIPTION
**Pull request by floren**
Signed-off-by: John Floren <john.floren@gravwell.io>

### CODE REVIEW

File: build/build-usb.bash [hunk 1]
```diff
@@ -8,6 +8,9 @@ else
     echo sudo mount --bind $HARVEY_GOLANG ../sys/go
     sudo mount --bind $HARVEY_GOLANG ../sys/go
     ls -l ../sys/go
+    pushd ../sys/src/cmd/go
```
Comment (rminnich) [inline build/build-usb.bash:11]: (cd ../sys/src/cmd/go && sh fetchrepos.sh)
it's just a lot safer, a missing pushd/popd can make life awful.

**Review by rminnich — APPROVED** [review]
This is great. Ignore my comment if you'd rather.

**Comment by floren** [comment]
PTAL

**Review by rminnich — APPROVED** [review]

**Review by rminnich — APPROVED** [review]
I just built a usb stick with this, booted as a terminal, and it works wonderfully well on my coreboot x230!

It's nice to have go there and "just work" and there's edwood too! And git!

**Pull request merged**